docs/about/contributing.md

```diff
@@ -49,9 +49,16 @@ to the git repository.
 
 ## Running the tests
 
-To run the tests, it is recommended that you use [Tox]. This just needs
-to be pip installed and then the test suite can be ran for MkDocs but running
-the command `tox` in the root of your MkDocs repository.
+To run the tests, it is recommended that you use
+[Tox](https://pypi.org/project/tox/).
+
+Install Tox using [pip](https://pypi.org/project/pip/) by running the
+following command:
+```
```

Comment:
As reported by the [markdownlint][1] test, the fenced code block should be surrounded by blank lines and have a language defined (use 'bash')

[1]: https://travis-ci.org/mkdocs/mkdocs/jobs/511133946#L510